[2025-03-14T14:01:45.893+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-14T14:01:45.947+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_prep_transformation_model_building.data_validation manual__2025-03-14T14:01:19.123476+00:00 [queued]>
[2025-03-14T14:01:45.982+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_prep_transformation_model_building.data_validation manual__2025-03-14T14:01:19.123476+00:00 [queued]>
[2025-03-14T14:01:45.984+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-03-14T14:01:46.091+0000] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): data_validation> on 2025-03-14 14:01:19.123476+00:00
[2025-03-14T14:01:46.187+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=354) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-14T14:01:46.169+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'data_prep_transformation_model_building', 'data_validation', 'manual__2025-03-14T14:01:19.123476+00:00', '--job-id', '242', '--raw', '--subdir', 'DAGS_FOLDER/data_prep_transformation_model_building.py', '--cfg-path', '/tmp/tmpvup35he2']
[2025-03-14T14:01:46.190+0000] {standard_task_runner.py:72} INFO - Started process 372 to run task
[2025-03-14T14:01:46.190+0000] {standard_task_runner.py:105} INFO - Job 242: Subtask data_validation
[2025-03-14T14:01:46.748+0000] {task_command.py:467} INFO - Running <TaskInstance: data_prep_transformation_model_building.data_validation manual__2025-03-14T14:01:19.123476+00:00 [running]> on host 0bbddc3bb474
[2025-03-14T14:01:47.194+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='data_prep_transformation_model_building' AIRFLOW_CTX_TASK_ID='data_validation' AIRFLOW_CTX_EXECUTION_DATE='2025-03-14T14:01:19.123476+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-14T14:01:19.123476+00:00'
[2025-03-14T14:01:47.200+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-14T14:01:47.872+0000] {logging_mixin.py:190} INFO - Data Validation Completed. Report saved at:
[2025-03-14T14:01:47.873+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-14T14:01:47.905+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-14T14:01:47.907+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=data_prep_transformation_model_building, task_id=data_validation, run_id=manual__2025-03-14T14:01:19.123476+00:00, execution_date=20250314T140119, start_date=20250314T140145, end_date=20250314T140147
[2025-03-14T14:01:48.258+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-14T14:01:48.336+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-14T14:01:48.363+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
